packages/contracts/src/dollar/interfaces/ICollateralOracle.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+/**
+ * @notice Collateral Oracle interface
+ * @notice Returns price quotes for all registered collaterals
+ */
+
+interface ICollateralOracle {
```

Comment:
There already exists an exact interface https://github.com/ubiquity/ubiquity-dollar/blob/dac1794b832a8883fa15794eb5afa7f4054680a8/packages/contracts/src/dollar/interfaces/ITWAPOracleDollar3pool.sol

Can't we keep only 1 of them?

Reply:
Good point, I already removed this in my local workspace.